GoMain/src/main/main.go

```diff
@@ -161,5 +165,28 @@ func orchestrationInit() error {
 
 	log.Println(logPrefix, "orchestration init done")
 
+	if isSecured {
+		mnedcmgr.GetServerInstance().SetCipher(dummy.GetCipher(cipherKeyFilePath))
+		mnedcmgr.GetServerInstance().SetCertificateFilePath(certificateFilePath)
+		mnedcmgr.GetClientInstance().SetCertificateFilePath(certificateFilePath)
+	} else {
+		mnedcmgr.GetServerInstance().SetCipher(sha256.GetCipher(cipherKeyFilePath))
+	}
+	isMNEDCServer := false
+	isMNEDCClient := false
+	if strings.Contains(buildTags, "mnedcserver") {
+		isMNEDCServer = true
+	} else if strings.Contains(buildTags, "mnedcsupport") {
```

Comment:
How about using "mnedcclient" instead of "mnedcsupport" for ease of understanding?

Reply:
Done